molgenis-data/src/main/java/org/molgenis/data/util/MetaUtils.java

```diff
@@ -32,22 +28,4 @@ public static Fetch getEntityTypeFetch() {
         .field(TAGS)
         .field(BACKEND);
   }
-
```

Comment:
We might want to consider removing MetaUtils altogether, move the remaining Fetch to EntityTypeUtils. Should be a seperate PR then I think.